src/lib/syncPagePath.ts

```diff
@@ -265,7 +298,9 @@ export default async function syncPagePath(this: APIv2PageUploadCommands) {
   const count = { succeeded: 0, failed: 0 };
 
   // topological sort the files
-  const sortedFiles = sortFiles((unsortedFiles as PageMetadata<PageRequestRepresentation>[]).sort(byParentPage));
+  const sortedFiles = sortFiles(
+    (unsortedFiles as PageMetadata<GuidesReferenceRequestRepresentation>[]).sort(byParentPage),
+  );
```

Comment:
can you do this instead ([derived from the work i did here](https://github.com/readmeio/rdme/pull/1272/files#diff-387a87252a58334e9a8eac46fabbe6a288d96043a28f79bf0f342d36ff826a7c))? that way it doesn't actually run the sort function for custom pages:

```suggestion
  const sortedFiles =
    this.route === 'custom_pages'
      ? (unsortedFiles as PageMetadata<CustomPagesRequestRepresentation>[])
      : sortFiles((unsortedFiles as PageMetadata<GuidesReferenceRequestRepresentation>[]).sort(byParentPage));
```